.travis.yml

```diff
@@ -76,13 +77,15 @@ matrix:
         - RAY_GCS_SERVICE_ENABLED=false
       install:
         - eval `python $TRAVIS_BUILD_DIR/ci/travis/determine_tests_to_run.py`
+        - if [ $RAY_CI_PYTHON_AFFECTED != "1" ]; then exit; fi
```

Comment:
for gcs_service test, we actually want to run it on every python changes

Reply:
Oh, so you mean, it should run even if only RAY_CI_RLLIB_AFFECTED=1?